Update docs on FQL builtins (#4374)
segmentio/segment-docs: src/api/public-api/fql.md

Destination Filter Reference documentation can be found in the [main Config API reference docs](https://reference.segmentapis.com/#6c12fbe8-9f84-4a6c-848e-76a2325cb3c5).

> Filter Query Language ("FQL") is a simple language for filtering JSON objects used by the Transformations API to conditionally apply transformations. In the Transformations API, FQL statements can either be used:

Reviewer: ```suggestion
The Transformations API uses Filter Query Language (FQL) to filter JSON objects and conditionally apply transformations. You can use FQL statements to:
```

Active voice

> 1. as filters that evaluate to `true` or `false` based on the contents of each Segment event. If the statement evaluates to `true`, the transformation is applied, and if it is `false` the transformation is not applied.

Reviewer: ```suggestion
- Apply filters that evaluate to `true` or `false` based on the contents of each Segment event. If the statement evaluates to `true`, the transformation is applied, and if it is `false` the transformation is not applied.
```

> 2. to [define new properties based on the result of an FQL statement](/docs/protocols/transform/#use-cases).

Reviewer: ```suggestion
- [Define new properties based on the result of an FQL statement](/docs/protocols/transform/#use-cases).
```
